observability/opentelemetry-metrics-backend/src/main/scala/sttp/client4/opentelemetry/OpenTelemetryMetricsBackend.scala

```diff
@@ -195,6 +211,32 @@ private class OpenTelemetryMetricsListener(
     b = config.description.fold(b)(b.setDescription)
     b.build()
   }
+
+  private def createRequestAttributes(request: GenericRequest[_, _]): Attributes = {
+    val attributes = Attributes
+      .builder()
+      .put(AttributeKey.stringKey("http.request.method"), request.method.method)
+      .put(AttributeKey.stringKey("server.address"), request.uri.host.getOrElse("unknown"))
+      .put(AttributeKey.longKey("server.port").asInstanceOf[AttributeKey[Any]], request.uri.port.getOrElse(80).toLong)
```

Comment:
can we use a simpler version without casting ? :thinking: 
`.put(AttributeKey.longKey("server.port"), request.uri.port.getOrElse(80))`

Reply:
Yeah! I just added typecast initially to look it's behavior and left it.